easy_my_coop_es_website/views/become_cooperator_view.xml

```diff
@@ -1,33 +1,75 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <data>
-        <template
-            id="become_cooperator_es"
-            inherit_id="easy_my_coop_website.becomecooperator"
-            name="Become cooperator"
+    <template id="es_iban_template" name="es_iban_template">
+        <div
+            class="form-group field-bank_account"
+            t-attf-class="form-group #{error and 'iban' in error and 'has-error' or ''}"
+            name="iban_container"
         >
-            <xpath expr="//label[@for='iban']/.." position="after">
-                <div
-                    t-attf-class="form-group #{error and 'vat' in error and 'has-error' or ''}"
-                >
-                    <label class="col-md-3 col-sm-4 control-label" for="vat">VAT</label>
-                    <div
-                        class="col-md-7 col-sm-8 bottom-line"
-                        style="padding-bottom:20px"
-                    >
-                        <input
-                            type="text"
-                            class="form-control"
-                            name="vat"
-                            placeholder="Y5643925S"
-                            required="True"
-                        />
-                    </div>
-                </div>
-            </xpath>
-            <xpath expr="//div[@name='share_div']" position="attributes">
-                <attribute name="hidden">1</attribute>
-            </xpath>
-        </template>
-    </data>
+            <label for="iban">Bank Account Number</label>
+            <input
+                class="form-control form-control-sm"
+                type="text"
+                name="iban"
+                id="iban"
+                required="True"
+                t-att-readonly="logged"
+                t-attf-value="#{iban or ''}"
+                placeholder="BE48523080767127"
```

Comment:
Can we use an IBAN with es format? `ES9000246912501234567891`

Reply:
The iban is valid, is it OK if I remove the last digit ? (`ES900024691250123456789`)